ports/boost-modular-build-helper/vcpkg.json

```
@@ -1,9 +1,13 @@
 {
   "name": "boost-modular-build-helper",
   "version": "1.77.0",
-  "port-version": 2,
+  "port-version": 3,
   "description": "Internal vcpkg port used to build Boost libraries",
   "dependencies": [
-    "boost-uninstall"
+    "boost-uninstall",
+    {
+      "name": "vcpkg-cmake",
+      "host": true
+    }
```

Because `boost-modular-build-helper` is a script helper, it should be depending on other scripts without `host` (since it's a script-to-script dependency, in order to allow `include("${CMAKE_CURRENT_LIST_DIR}/../vcpkg-cmake/vcpkg-port-config.cmake")` to work)

```suggestion
    "name": "vcpkg-cmake"
```

see [vcpkg-qmake](https://github.com/microsoft/vcpkg/blob/master/ports/vcpkg-qmake/vcpkg-port-config.cmake) as an example